### PR TITLE
Tx 2

### PR DESCRIPTION
**Pull request by osadolortech**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+db.sqlite3
+SECRET_KEY = django-insecure-)$h@de$2$hfjs5a#=wotr65!+h_uzfzl(0%yqv0a97+3^+@ba^
```
Comment (iamr0b0tx) [inline .env]: no space

File: .env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+db.sqlite3
```
Comment (iamr0b0tx) [inline .env:1]: use postgres and database url

File: blogapi/serializers.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+from .models import BlogpotslList
+
+
+class Blogserilizers(serializers.ModelSerializer):
```
Comment (iamr0b0tx) [inline blogapi/serializers.py]: spelling

File: blogapi/serializers.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    class Meta:
+        model = BlogpotslList
```
Comment (iamr0b0tx) [inline blogapi/serializers.py]: spelling

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]

File: Procfile [hunk 5]
```diff
@@ -0,0 +1,5 @@
+release: python manage.py makemigrations --no-input
```
Comment (iamr0b0tx) [inline Procfile:1]: this is bad

File: blogapi/views.py [hunk 6]
```diff
@@ -1,9 +1,12 @@
...
 from .models import BlogpotslList
 from .serializers import Blogserilizers
+from rest_framework.pagination import LimitOffsetPagination
 
 # Create your views here.
 
 class BlogApi(viewsets.ModelViewSet):
     queryset = BlogpotslList.objects.all()
     serializer_class = Blogserilizers
+    pagination_class = LimitOffsetPagination
```
Comment (iamr0b0tx) [inline blogapi/views.py]: you don't need this. the default should work without this line

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]

File: Procfile (hunk 5 above)
Comment (osadolortech) [reply Procfile:1]: can i remove the line sir? will it still work?

Comment (iamr0b0tx) [reply Procfile:1]: I said its bad, understand what that means

File: .env (hunk 1 above)
Comment (osadolortech) [reply .env]: fixed

File: .env (hunk 2 above)
Comment (osadolortech) [reply .env:1]: fixed

File: blogapi/serializers.py (hunk 3 above)
Comment (osadolortech) [reply blogapi/serializers.py]: fixed

File: blogapi/serializers.py (hunk 4 above)
Comment (osadolortech) [reply blogapi/serializers.py]: fixed

File: blogapi/views.py (hunk 6 above)
Comment (osadolortech) [reply blogapi/views.py]: fixed

**Pull request merged**